db/migrate/20191027182229_create_players.rb

```diff
@@ -0,0 +1,23 @@
+class CreatePlayers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :players do |t|
+      t.string :active
+      t.string :jersey
+      t.string :lname
+      t.string :fname
+      t.string :display_name
+      t.string :team
+      t.string :position
+      t.string :height
+      t.string :weight
+      t.string :college
+      t.string :experience
+      t.string :birth_date
+      t.string :photo_url
+      t.string :bye_week
+      t.string :ffn_id
+      t.string :ffn_id 
```

Comment:
Do we want all these attributes to be strings? Is that the easiest way to deal with them?

Reply:
I don't think so actually. Numbers should either be integers or floats. You can pass a string in and it will get converted automatically if it can

Reply:
Maybe we want to specify those here then too.